src/applications/pensions/config/chapters/04-household-information/dependentChildInformation.js

```diff
@@ -0,0 +1,175 @@
+import get from '@department-of-veterans-affairs/platform-forms-system/get';
+import merge from 'lodash/merge';
+import moment from 'moment';
+
+import {
+  radioSchema,
+  radioUI,
+  ssnSchema,
+  ssnUI,
+  yesNoSchema,
+  yesNoUI,
+} from '@department-of-veterans-affairs/platform-forms-system/web-component-patterns';
+
+import fullSchemaPensions from 'vets-json-schema/dist/21P-527EZ-schema.json';
+import createHouseholdMemberTitle from '../../../components/DisclosureTitle';
+
+import {
+  dependentSeriouslyDisabledDescription,
+  dependentWarning,
+  disabilityDocs,
+  schoolAttendanceWarning,
+} from '../../../helpers';
+
+const { dependents } = fullSchemaPensions.properties;
+
+function isBetween18And23(childDOB) {
+  return moment(childDOB).isBetween(
+    moment()
+      .startOf('day')
+      .subtract(23, 'years'),
+    moment()
+      .startOf('day')
+      .subtract(18, 'years'),
+  );
+}
+
+// Checks to see if they’re under 17.75 years old
+function isEligibleForDisabilitySupport(childDOB) {
+  return moment()
+    .startOf('day')
+    .subtract(17, 'years')
+    .subtract(9, 'months')
+    .isBefore(childDOB);
+}
+
+/** @type {PageSchema} */
+export default {
+  uiSchema: {
+    dependents: {
+      items: {
+        'ui:title': createHouseholdMemberTitle('fullName', 'Information'),
+        childPlaceOfBirth: {
+          'ui:title': 'Place of birth (city and state or foreign country)',
+        },
+        childSocialSecurityNumber: merge({}, ssnUI(), {
+          'ui:required': (formData, index) =>
+            !get(`dependents.${index}.view:noSSN`, formData),
+        }),
+        'view:noSSN': {
+          'ui:title': "Doesn't have a Social Security number",
+        },
+        childRelationship: radioUI({
+          title: "What's your relationship?",
```

Comment:
I don't think that including the dependent's full name in this title should be a requirement since it's still in the context of a full page with the dependent's name in the header. When we cut it over to it's own page, we can spend time updating the title.